### ISSUE TITLE
HLC from Toadie

### ISSUE THREAD

**Issue by nicolasbadano** [KoffeinFlummi/AGM_Compatibility#6]
**Status**: Golden, 9/10

---

http://forums.bistudio.com/showthread.php?172850-Toadie-s-SmallArms-and-Animations-for-Arma3
- [x] Bullet Mass
- [x] Barrel swap
- [x] Jam Chance
- [x] Bipods
- [ ] Scopes

**Comment by jonpas** [comment]
HLC M60E4 Changelog: "Added support to Bipod functionality and Overheating/Barrel Swapping support for AGM"

And many other new weapons added and changes made, no idea what must be added/removed, @esteldunedain you would know best probably. :)

**Comment by esteldunedain** [comment]
OK. jonpas, I assume toadie made a new release; can you contact him and ask him to include this compatiblity stuff?

Maybe he can take over and there's no more need for this pbo.

**Comment by jonpas** [comment]
That's great idea, contacting him now.

**Comment by esteldunedain** [comment]
@jonpas, we always meant these compat pbos to kickstart work for addon makers themselves. If the authors want to take over, it's much better for everyone. Besides, most things (all the things in case of toadie), are just additional entries that are ignored when not using AGM, so there's no harm in including them on the base addons.

**Comment by jonpas** [comment]
Yup I understand that it causes no harm and is way better (it basically means the modder can quickly add AGM stuff as well right next to his other additions), I just slightly forgot we could just ask him to implement it. :)

I've sent him a nice PM if he is willing to implement the config entries on this project natively into his mod, I believe he will as for AUGs he already implemented Barrel Swapping/Overheating.
